src/services/paras/ParasService.ts

```diff
@@ -258,8 +262,11 @@ export class ParasService extends AbstractService {
 			winning = null;
 		}
 
+		const leasePeriodsPerSlot =
```

Comment:
Assuming this will be `8` in the future, and there will be no need for `LEASE_PERIODS_PER_SLOT_FALLBACK`?

Reply:
Yup exactly, check line 35